packages/frontend-shared/src/locales/en-US.json

```diff
@@ -66,14 +66,14 @@
       "fasterTests": "Run tests faster",
       "smartOrchestration": "Smart Orchestration",
       "prompts": {
-        "ci": {
+        "ci1": {
```

Comment:
Initially it was to be consistent with the existing implementation, which does seem to account for more than just the basic set of 2 prompts we have now. I decided to keep it because I could see stuff like this getting A/B tested down the line, which would introduce more variations, and it seems a little more clear to me if we imply that from the outset.